examples/dataview/formula-data-view-all-modes.html

```diff
@@ -161,9 +160,9 @@
       const formulaLocalElement = document.getElementById('formula-local');
       const errorContainer = document.getElementById('errorContainer');
 
-      let formulaRemote;
-      let formulaRemoteFiltered;
-      let formulaLocal;
+      let formulaGlobal;
+      let formulaViewportPrecise;
```

Comment:
Same here, II would just call the example 'formulaDataView' and remove the viewport variants, leaving just the default

Reply:
I see now we have another one with that name. I guess we could mix them both together, having 'global' and 'viewport'